packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -192,7 +193,19 @@ export interface TableOptions {
 
   /**
    * The name of TTL attribute.
-   * @default - TTL is disabled
+   *
+   * If a value is passed a custom resource is created. The custom resource does add support
+   * for changes to the TTL attribute later.
+   *
+   * If the TTL should be removed from the table, you should pass an empty string (e.g., '').
+   *
+   * If you wish to remove the custom resource, you can leave the default (undefined). However,
+   * note that this will leave the table in the current state. A new table will not have a disabled
+   * TTL, while an enabled TTL will remain in place if you defined one before.
+   *
+   * A 'Time to live has been modified multiple times within a fixed interval' error can occur if
+   * you recently changed the ttl. In this case wait some time and try again.
+   * @default - TTL stays in the current state (disabled for newly created tables).
```

Comment:
Let's leave the default docs as they are now.

Reply:
I have reverted the `@default` text.